### ISSUE TITLE
Could not load `import xxx from '@/utils/xxx`

### ISSUE THREAD

**Issue by tghkuma** [cdmoro/bootstrap-vue-3#463]
The following error in `yarn build`

```
 ERROR  [vite:load-fallback] Could not load /project_path/utils/getBreakpointProps (imported by ../../node_modules/bootstrap-vue-3/src/components/BCol.vue): ENOENT: no such file or directory, open '/project_path/utils/getBreakpointProps'


 ERROR  Could not load /project_path/utils/getBreakpointProps (imported by ../../node_modules/bootstrap-vue-3/src/components/BCol.vue): ENOENT: no such file or directory, open '/project_path/utils/getBreakpointProps'
 `````

Shouldn't it be specified as a relative path, as pointed out in #5?

**Comment by VividLemon** [comment]
I'm not sure how you're doing this but I have no issue with a test library on building and/or running dev with the latest version

**Comment by tghkuma** [comment]
@VividLemon It may work if it is defined in `tsconfig.json`-> compilerOptions-> paths.
However, `npm install bootstrap-vue-3` does not include` tsconfig.json`.

**Comment by VividLemon** [comment]
Should be fixed in 0.1.16
